src/Mod/BIM/Arch.py

```diff
@@ -478,7 +478,7 @@ def makeProfile(profile=[0,'REC','REC100x100','R',100,100]):
         FreeCAD.Console.PrintError("No active document. Aborting\n")
         return
     obj = FreeCAD.ActiveDocument.addObject("Part::Part2DObjectPython",profile[2])
```

Comment:
IMO it would make sense to also append "_" to the object name.

Reply:
Thanks for the review! If I understand this correctly, that would mean changing the name from e.g. `Beam001` to `Beam_001`. While I personally favor this naming scheme, it would be inconsistent with how objects with identical labels across other FreeCAD modules have the numerical suffix added to make them unique (`Body001`, `Array001`, etc.). For this PR, I'd rather stick to the minimal changes to address the original bug, thus I'll mark the conversation as resolved. Feel free to reopen if you see this differently, or in case I did not interpret the review comment correctly.

Reply:
Agree with @furgo16 here, also because the object name is indeed something more "basic" like "Profile" or "Beam" and not specific like "Profile 300x200" like the label, it's more an identifier, and the less fancy characters the better IMHO

Reply:
We are talking about the profile here, not about the beam or whatever else that is created from it.

For the 22x100 profile (Eurocode timber) these object names are now used:
_2x100
_2x101
_2x102
etc.

Note that there is a 2nd issue: the first digit is replaced.
Not desirable IMO.

Line 480 should probably be:
obj = FreeCAD.ActiveDocument.addObject("Part::Part2DObjectPython", "P_" + profile[2] + "_")

Reply:
> For the 22x100 profile (Eurocode timber) these object names are now used

Oh indeed that's not desirable, sorry, I hadn't looked any further. I suggest we use this, then:
`obj = FreeCAD.ActiveDocument.addObject("Part::Part2DObjectPython", "Profile")`
and then only the label is set to the profile name

Reply:
```suggestion
    obj = FreeCAD.ActiveDocument.addObject("Part::Part2DObjectPython", "Profile")
```

Sounds good, thanks both. Adding and applying the suggestion.